### PR TITLE
Add a post announcing the second step of the jbuilder deprecation

### PR DESCRIPTION
**Pull request by diml**

### CODE REVIEW

File: _posts/2019-01-30-second-step-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+discuss: https://discuss.ocaml.org/t/second-stage-of-the-jbuilder-deprecation
+---
+
+As [planned][migration], we are now entering the second stage of the
+Jbuilder deprecation. This means that the `jbuilder` binary still
+exists and work as before however it prints a warning on every
+startup. Additionally, `jbuild` files are still accepted but both
+`jbuilder` and `dune` display a warning when encountering them. This
+behavior is now in the development version of dune and will be part of
+the upcoming 1.7 release.
```
Comment (emillon) [inline _posts/2019-01-30-second-step-deprecation.md]:
```suggestion
the upcoming 1.7.0 release.
```

File: _posts/2019-01-30-second-step-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+The support for the `jbuilder` binary and `jbuild` files will be
+discontinued in July this year. If you haven't switch your project to
```
Comment (emillon) [inline _posts/2019-01-30-second-step-deprecation.md]: I think it's better to be explicit here:

```suggestion
discontinued in July 2019. If you haven't switch your project to
```

File: _posts/2019-01-30-second-step-deprecation.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+dune already, now is a good time to do it.
+
+To do that, you can either follow the [migration's guide][migration],
+or use the newly added `dune upgrade` command that does it for you.
+
+Using the automatic upgrader
+----------------------------
+
+The automatic upgrader is a new feature in dune 1.7. It automatically
```
Comment (emillon) [inline _posts/2019-01-30-second-step-deprecation.md]:
```suggestion
The automatic upgrader is a new feature in dune 1.7.0. It automatically
```

File: _posts/2019-01-30-second-step-deprecation.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+```
+
+Interaction with git
+--------------------
+
+The upgrader detects when it is running inside a git repository. In
+such cases, it issues git commands in such a way that running `git
+diff` just after `dune upgrade` will show you exactly what changed. If
```
Comment (emillon) [inline _posts/2019-01-30-second-step-deprecation.md]: What commands does it perform? As I understand it, even if it was git agnostic, `git diff` would already display what changed.

File: _posts/2019-01-30-second-step-deprecation.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+you do no like the result or notice a problem with the automatic
```
Comment (emillon) [inline _posts/2019-01-30-second-step-deprecation.md]:
```suggestion
you do not like the result or notice a problem with the automatic
```

File: _posts/2019-01-30-second-step-deprecation.md (hunk 4 above)
Comment (diml) [reply _posts/2019-01-30-second-step-deprecation.md]: It does things such as `git mv jbuild dune`. If we didn't do that, then `git diff` would just show the `jbuild` file as deleted and would ignore the new `dune` file

Comment (emillon) [reply _posts/2019-01-30-second-step-deprecation.md]: Ah, didn't think of this :)

**Pull request merged**